src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -513,13 +492,7 @@ protected override async Task<bool> AcceptCommand(MessageId id, JObject parms, C
                 case "Debugger.setPauseOnExceptions":
                     {
                         string state = args["state"].Value<string>();
-                        context.PauseOnExceptions = state switch
-                        {
-                            "all"      => PauseOnExceptionsKind.All,
-                            "uncaught" => PauseOnExceptionsKind.Uncaught,
-                            "none"     => PauseOnExceptionsKind.None,
-                            _          => PauseOnExceptionsKind.Unset
-                        };
+                        context.PauseOnExceptions = GetPauseOnExceptionsStatusFromString(state);
```

Comment:
Should `context.PauseOnExceptions` be set if `GetPauseOnExceptionsStatusFromString` returns `.Unset`? It would override the existing value that it would have. Same for the other location.